source/_docs/create-custom-upstream.md

```diff
@@ -1,19 +1,78 @@
 ---
-title: Create a Custom Upstream
+title: Early Access: Create a Custom Upstream
 description: Connect a remote repository with Pantheon to use as a starting point for new sites.
 tags: [tools, workflow]
-categories: []
+earlyaccess: true
+earlynote: The documentation on this page discusses features and options that are not available across the entire platform.
 ---
+Pantheon is rolling out a new self-serve feature to support creating Custom Upstreams in the Organization Dashboard. Once created, members of the organization will be able to create new sites from the common codebase. For an overview of this feature, see [Introduction to Custom Upstreams](/docs/custom-upstream).
 
-The following steps will help you get a Custom Upstream ready for prime time within your organization. Once completed, members of the organization will be able to create new sites from the common codebase. For an overview of this feature, see [Introduction to Custom Upstreams](/docs/custom-upstream).
+## Eligibility
+Access to this feature is currently invite only for Agency and Enterprise Organizations. Repositories must be hosted on [GitHub](https://github.com/) or [BitBucket](https://bitbucket.org/). If you would like early access to this feature, or if you need to use an alternate repository hosting provider, please let us know.
+
```

Comment:
Suggestion for clarity:

"Contact us for early access to this feature. If you need to use an alternate repository hosting provider, let us know in the form."